README.md

```diff
@@ -16,6 +16,7 @@ I renamed the repository to "Awesome Alternatives in Rust". The original name wa
   - [Text editors](#text-editors)
   - [Text processing](#text-processing)
   - [Utilities](#utilities)
+  - [Performance](#performance)
```

Comment:
Could you follow alphabet order? In this case, `Performance` should be added between `Container` and `System tools`.

Reply:
Done